test/basic_test.rb

```diff
@@ -124,6 +124,16 @@ def test_arel_visit_RGeo_ActiveRecord_SpatialNamedFunction_with_distinct
     assert_equal("SPATIAL_FUNC(DISTINCT ST_GeomFromText('POINT (1.0 2.0)'), ST_GeomFromText('LINESTRING (1.0 2.0, 2.0 3.0)'))", sql.value)
   end
 
+  def test_multi_geom_sanitization
+    multi_geom = RGeo::Geos.factory.parse_wkt("MULTIPOLYGON (((0 0, 0 1, 1 1, 0 0)),((1 1, 0 0, 0 1, 1 1)))")
+    sql = "ST_DWithin(geom, :geom, :buffer)"
```

Comment:
why do we need that `st_dwithin` function for the test?

Reply:
We do not need it.  It could be SQL valid or not that has a geom interpolated.

Reply:
I think I'd keep it simple then :)